frontend/src/components/Modals/Password/MultiUserAuth.jsx

```diff
@@ -29,66 +29,59 @@ export default function MultiUserAuth() {
 
   return (
     <form onSubmit={handleLogin}>
-      <div className="relative bg-white rounded-lg shadow dark:bg-stone-700">
-        <div className="flex items-start justify-between p-4 border-b rounded-t dark:border-gray-600">
+      <div
+        className="flex flex-col justify-center items-center relative bg-white rounded-2xl shadow dark:bg-stone-700 border-2 border-slate-300 border-opacity-20 w-[400px]"
+        style={{
+          background:
+            "linear-gradient(180deg, rgba(61, 65, 71, 0.30) 0%, rgba(44, 47, 53, 0.30) 100%)",
+          boxShadow: "0px 4px 30px rgba(0, 0, 0, 0.25)",
```

Comment:
If we reuse these classes - make them a custom class in the index.css and add it to classNames. If we ever want to touch the gradient we will be in for max pain changing inline everywhere